tuning/scripts/master_tuning_script.sh

```diff
@@ -1,414 +1,330 @@
-        sudo wget https://github.com/google/googletest/archive/release-1.10.0.tar.gz  && \
-        sudo tar -xvf release-1.10.0.tar.gz  && \
-        pushd googletest-release-1.10.0 && \
-        sudo mkdir build && pushd build && sudo cmake .. && sudo make && sudo make install && popd && popd && popd
-    fi
+WORKING_PATH=${1}
+LOG_PATH=${2}
 
-    # Install Lapack
-    if [[ $(ls -A /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.7.1 | wc -c) -eq 0 ]]; then
-        sudo mkdir -p /usr/src/lapack && pushd /usr/src/lapack && \
-        sudo wget https://github.com/Reference-LAPACK/lapack-release/archive/lapack-3.7.1.tar.gz  && \
-        sudo tar -xvf lapack-3.7.1.tar.gz  && \
-        pushd lapack-release-lapack-3.7.1 && \
-        sudo mkdir build && pushd build && \
-        sudo cmake .. -DCBLAS=ON -DLAPACKE=OFF -DBUILD_TESTING=OFF -DCMAKE_Fortran_FLAGS='-fno-optimize-sibling-calls' && \
-        sudo make && sudo make install && popd && popd && popd
-    fi
+if [ -z ${TENSILE_PATH} ]; then
+  TENSILE_PATH=${WORKING_PATH}/tensile/Tensile
 fi
 
-if [[ "${HCC}" == true ]]; then
-    TENSILE_COMPILER=hcc
-    CODE_OBJECT_VERSION=V2
-    ROCBLAS_COMPILER=no-hip-clang
-    export PATH=${ROCM_PATH}/bin:${ROCM_PATH}/hip/bin:${ROCM_PATH}/hcc/bin:${PATH}
-else
-    TENSILE_COMPILER=hipcc
-    CODE_OBJECT_VERSION=V3
-    ROCBLAS_COMPILER=hip-clang
-    export PATH=${PATH}:${ROCM_PATH}/bin:${ROCM_PATH}/hip/bin:${ROCM_PATH}/llvm/bin
+if [ -z ${ROCBLAS_PATH} ]; then
+  ROCBLAS_PATH=${WORKING_PATH}/rocblas/rocBLAS
 fi
 
-if [[ "${DATA_TYPE}" == hgemm || "${DATA_TYPE}" == h ]]; then
-    DATA_TYPE=hgemm
-    DVAL=4
-elif [[ "${DATA_TYPE}" == dgemm || "${DATA_TYPE}" == d ]]; then
-    DATA_TYPE=dgemm
-    DVAL=1
-else
-    if [[ "$(grep -c 'r s' ${LOG})" -gt 0 || "$(grep -c 'r f32' ${LOG})" -gt 0 || "$(grep -c 'sgemm' ${LOG})" -gt 0 || "$(grep -c 'a_type f32' ${LOG})" -gt 0 || "$(grep -c '"a_type: "f32_r"' ${LOG})" -gt 0 ]]; then
-        printf "sgemm detected\n"
-        DATA_TYPE=sgemm
-        DVAL=2
-    elif [[ "$(grep -c 'r d' ${LOG})" -gt 0 || "$(grep -c 'r f64' ${LOG})" -gt 0 || "$(grep -c 'dgemm' ${LOG})" -gt 0 ]]; then
-        printf "dgemm detected\n"
-        DATA_TYPE=dgemm
-        DVAL=1
-    elif [[ "$(grep -c 'r h' ${LOG})" -gt 0 || "$(grep -c 'r f16' ${LOG})" -gt 0 || "$(grep -c 'hgemm' ${LOG})" -gt 0 || "$(grep -c 'a_type f16' ${LOG})" -gt 0 || "$(grep -c 'a_type: "f16_r"' ${LOG})" -gt 0 ]]; then
-        printf "hgemm detected\n"
-        DATA_TYPE=hgemm
-        DVAL=4
-        if [[ "$(grep -c 'compute_type f16_r' ${LOG})" -gt 0 || "$(grep -c 'compute_type: "f16_r"' ${LOG})" -gt 0 ]]; then
-            printf "compute_type of f16 detected, disabling HighPrecisionAccumulate\n"
-            DISABLE_HPA=true
-        fi
-    else
-        printf "Could not detect data type in log file, assuming sgemm\n"
-        DATA_TYPE=sgemm
-        DVAL=2
-    fi
+if ${DEPENDENCIES}; then
```

Comment:
Hi Ben, are you using tabs or spaces for your indentations?
I think the preferred formatting is 4 spaces for indent. Can you please update this in your editor?

I see a few inconsistencies above using 2 spaces, then here looks like tab.

Reply:
Also, did you also run .githooks/install in the Tensile repo root?

Reply:
Oops, look like I missed this file when I was cleaning up the indentation. 

I went for 2 spaces for these scripts as that's what they were before, but I can easily update them to 4 spaces if that is preferred.

Reply:
I think we will be shooting for 4 spaces moving forward. Does a lot for readability.